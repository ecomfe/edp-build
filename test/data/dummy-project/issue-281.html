<!doctype html>
<html>
  <head>
    <style type="text/css">
    .e { background: url(../../src/e.png); }
    </style>

    <style type='text/css'>
    .c { background: url(../../src/c.png); }
    </style>

    <STYLE type='text/css' data-my-key="my-value">
    .b { background: url(../../src/b.png); }
    div { color: red }
    </style>

    <style>
    .a { background: url(../../src/a.png); }
    </style>
  </head>
  <body>
    <STYLE>
    .a { background: url(../../src/a.png); }
    </style>
  </body>
</html>
