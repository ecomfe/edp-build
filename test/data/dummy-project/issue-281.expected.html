<!doctype html>
<html>
  <head>
    <style type="text/css">
    .e { background: url(../../asset/e.png); }
    </style>

    <style type='text/css'>
    .c { background: url(../../asset/c.png); }
    </style>

    <STYLE type='text/css' data-my-key="my-value">
    .b { background: url(../../asset/b.png); }
    div { color: red }
    </style>

    <style>
    .a { background: url(../../src/a.png); }
    </style>
  </head>
  <body>
    <STYLE>
    .a { background: url(../../src/a.png); }
    </style>
  </body>
</html>
